data/md/legislature/Nino-Mangione-346c3339-2f88-4d95-9888-546174d081fb.yml

```diff
@@ -0,0 +1,15 @@
+id: ocd-person/346c3339-2f88-4d95-9888-546174d081fb
+name: Nino Mangione
+given_name: Nino
+family_name: Mangione
+image: http://mgaleg.maryland.gov/2023RS/images/mangione01.jpg
+party:
+- name: Republican
```

Comment:
@sroomf this person should have a file already and just needs added:
- type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:md/government
  district: 42A
they previously were in house 42b from jan 2019 until 01/11/2023.